test/lit.cfg

```diff
@@ -135,7 +135,8 @@ if len(kleaver_extra_params) != 0:
 # Set absolute paths and extra cmdline args for KLEE's tools
 subs = [ ('%kleaver', 'kleaver', kleaver_extra_params),
   ('%klee','klee', klee_extra_params),
-  ('%ktest-tool', 'ktest-tool', '')
+  ('%ktest-tool', 'ktest-tool', ''),
+  ('%klee-replay', 'klee-replay', '')
```

Comment:
It does invoke `klee-replay`, I checked this.  Is only when metaSMT is used that this happens, not sure why.   I'll change the order, thanks.